cert/lib/cert/options.rb

```diff
@@ -60,7 +60,17 @@ def self.available_options
                                      env_name: "CERT_KEYCHAIN_PASSWORD",
                                      sensitive: true,
                                      description: "This might be required the first time you access certificates on a new mac. For the login/default keychain this is your account password",
-                                     optional: true)
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :platform,
+                                     env_name: "CERT_PLATFORM",
+                                     description: "Set the provisioning profile's platform (ios, macos)",
+                                     is_string: false,
```

Comment:
FYI the suggestion from @KrauseFx  was followed in commit 6df6ff1d4b29b88382e0d0be1ad5c56a133afb58 and it is currently a symbol for `sigh`.